bookwyrm/templates/feed.html

```diff
@@ -47,7 +47,17 @@ <h2 class="title is-5">Your books</h2>
                 </div>
             </div>
             <div class="card-content">
-                {% include 'snippets/shelve_button.html' with book=book %}
+                <div class="columns is-gapless">
```

Comment:
I think this works better just flat, rather than in columns -- so this markup minus all the added divs? it was too long for my screen as is

Reply:
I ended up flattening this and then totally reworking the update snippet. Curious to hear what you think, I'm...mostly happy with it I think :sweat_smile: